lib/scarpe/internal_app.rb

```diff
@@ -21,12 +21,13 @@ def render(&block)
     end
 
     def append(el)
-      @window.eval("document.getElementById(#{current_id}).insertAdjacentHTML('beforeend', \`#{el}\`)")
+      @window.eval("document.getElementById(#{current_id}).insertAdjacentHTML('beforeend', \"#{el}\")")
```

Comment:
I think will break some rendering (when `el` contains `"`)
`\`` is used in javascript for strings that support interpolation and should have less overlap with general dom elements

Reply:
Sadly I don't recall which example caused this to fail, most likely one of the button ones